apps/calendar/elements/modify_event.html

```diff
@@ -79,14 +79,25 @@ <h3 data-l10n-id="remind-me" role="heading" aria-level="2">Remind me</h3>
 
       </ol>
       <div class="delete">
-        <a role="button" data-l10n-id="event-delete" class="danger delete-record bb-button">
+        <a data-l10n-id="event-delete" role="button" href="#remove-event-dialog" class="danger delete-record bb-button">
           Delete Event
         </a>
       </div>
+    </form>
+
+    <section role="status">
+      <p class="errors"></p>
+    </section>
 
-      <section role="status">
-        <p class="errors"></p>
+    <form role="dialog" data-type="confirm" id="remove-event-dialog">
+      <section>
+        <h1 data-l10n-id="remove-event-dialog">Remove Event</h1>
+        <p data-l10n-id="remove-event-dialog-details">Are you sure you want to delete this event?</p>
```

Comment:
moving forward we should not add English copy to the markup.. everything should be handled by the `[data-l10n-id]` (seen some patches on other apps just removing the English words from markup..)